bindings/rust/evmc-declare-tests/src/lib.rs

```diff
@@ -16,7 +17,12 @@ impl EvmcVm for FooVM {
         FooVM {}
     }
 
-    fn execute(&self, _code: &[u8], _context: &ExecutionContext) -> ExecutionResult {
+    fn execute(
+        &self,
+        _code: &[u8],
```

Comment:
I figured out recently that the identifiers for the struct members and the arguments won't conflict, therefore we don't need to prefix with an underscore.

Reply:
These are in examples and are prefixed for making clippy happy since they are not used.